### PR TITLE
Computer noises

### PR DESCRIPTION
**Pull request by nmajask**
# Document the changes in your pull request

Partially ports ["Adds a bunch of looping audio to computers, gravegen and telecomms." #54324](https://github.com/tgstation/tgstation/pull/54324) by Qustinnus I think, only adds the bootup and shutdown sounds for mod PCs as I couldn't get the others to work. Also adds sounds from ["Adds PDA clicking sounds" #25522](https://github.com/Baystation12/Baystation12/pull/25522) and ["Adds keystrokes sounds" #19230](https://github.com/Baystation12/Baystation12/pull/19230) by comma on Baystation, as well as some from another PR on Bay that I can't find. Plays most of the time when you interact with mod PCs. There are a few issues with the bootup sound cutting off abruptly and the interaction sound not triggering when doing stuff that don't leave TGUI.

Basically, this adds a bootup and shutdown sound using sounds from TG and interaction sounds from Bay to mod PCs.

Will add examples later when I get the chance.

# Wiki Documentation

None needed. 

# Changelog

:cl:  Qustinnus, comma, and nmajask
soundadd: added a bootup and shutdown sound using sounds from TG and interaction sounds from Bay to mod PCs
/:cl:

### CODE REVIEW

**Comment by maxion12345** [comment]
should add fan noises to AI mainframes

**Comment by nmajask** [comment]
> should add fan noises to AI mainframes

I think you got the wrong person, I exclusively make changes to mod PCs, fix the alt titles PR, and make controversial nerfs. If someone were to tackle that, I would recommend porting the rest of the ambience sounds as it basically does that for tcomms, though when I tried porting it tcomms sounded like it was having a firefight.

**Review by JamieD1 — APPROVED** [review]

File: code/modules/modular_computers/file_system/programs/borg_monitor.dm [hunk 1]
```diff
@@ -131,9 +131,11 @@
 				return
 			if(R.stat == DEAD) //Dead borgs will listen to you no longer
 				to_chat(usr, "<span class='warn'>Error -- Could not open a connection to unit:[R]</span>")
+      computer.play_interact_sound()
```
Comment (JamieD1) [inline code/modules/modular_computers/file_system/programs/borg_monitor.dm]:
```suggestion
			computer.play_interact_sound()
```
